.github/scripts/mr_generate_summary.py

```diff
@@ -0,0 +1,65 @@
+# Generate a file with a summary of test, the file will publish as an artifact
```

Comment:
Can we add a bit more information what this is doing? E.g. something in this direction:

```suggestion
# Collect the results of the various model test runs which are done as part of the model regression CI pipeline and dump them as a single file artifact. This artifact will the then be published at the end of the tests.
```